print/nuxt.config.js

```diff
@@ -164,10 +164,11 @@ export default {
   },
 
   /**
-   * Environment variables available in application
+   * Environment variables available in the server at runtime
    */
-  env: {
-    FRONTEND_URL: process.env.FRONTEND_URL || 'http://localhost:3000',
```

Comment:
I'm actually not even sure, if we need the FRONTEND_URL variable at all anymore. I think in the past it was used to avoid CORS-errors and iFrame-embdding errors due to the fact that we used to have different subdomains for the various services. Not that everything is on the same domain, that might not be needed anymore. But we can clean this up in a separate PR.